integrations/docker/run.sh

```diff
@@ -35,7 +35,7 @@ ORG=${DOCKER_RUN_ORG:-connectedhomeip}
 IMAGE=${DOCKER_RUN_IMAGE:-$(basename "$here")}
 
 # version
-VERSION=${DOCKER_RUN_VERSION:-$(cat "$here/version")} ||
+VERSION=${DOCKER_RUN_VERSION:-latest} ||
```

Comment:
I believe this is on purpose since latest requires a separate update from master (i.e. we may have a version commited and not yet tagged as 'latest'.

I would drop this particular change,

Reply:
The cirque test will fail without this change, see similar comments https://github.com/project-chip/connectedhomeip/pull/3263#discussion_r508943922.

Reply:
see #3361 

push (all) the images first for now

Reply:
if you rebase, this is addressed on master with an updated version file in the cirque image directory

Reply:
> if you rebase, this is addressed on master with an updated version file in the cirque image directory

Thanks!